.env.sample

```diff
@@ -30,7 +30,7 @@ KAFKA_GROUP_ID = "projects"
 SUBMISSION_TOPIC = "dev.sl.projects.submissions"                                // Kafka topic name for pushing projects submissions
 PROJECT_SUBMISSION_TOPIC = "dev.sl.projects.submissions"                        // project submission topic 
 
-# SUNBIRD LOCATION AND USER READ
+# ELEVATE LOCATION AND USER READ
 USER_SERVICE_URL = "http://user-service:3000"                                   // service used for user profile read location search are using this base url
```

Comment:
@Prajwal17Tunerlabs add entity service url as entity and user service is different